### DISCUSSION TITLE
Is it possible to use private git as customers support forum?

### DISCUSSION THREAD

**Discussion by danyj** [community/community#21532] [New to GitHub]
<p>First kudos to this forum, it was long time coming.   </p>
<p>I have private git that I would love to use for my customers as a actual support forum. The process should be automated for example; </p>
<p>Customer logs in my CMS ( WP , Drupal, Joomla )  and they are automatically allowed to see and post in my private git.  As is right now we must send invites to anyone we want to give access to.  </p>
<p>Is something like this already available? </p>

**Comment by nadiajoyce** [comment, marked as answer]
<p><a class="mention" href="/danyj">@danyj</a> This isn’t something that GitHub offers as a feature at this time. It’s something that you *might* be able to implement with a lot of API work, which is something you might have more help with over on the <a href="https://platform.github.community/" rel="nofollow noopener">Platform Forum</a>. That said, there are a lot of potential complications with this. Two of the big ones that come to mind are: </p>
<ul>
<li>Every customer you have would have to have a GitHub account already in order to access and use that private repository</li>
<li>If the private repository you’re trying to direct them to exists in an organization account, you’d end up paying for a seat every time a customer got access to that private repository</li>
</ul>
<p>All in all, I think this is a really great idea, but it’s not something that exists out-of-the-box on <a href="http://github.com" rel="nofollow noopener">github.com</a> right now. I will, however, note it in our internal feature request list. Though I can’t guarantee anything or share a timeline for if or when this might be implemented, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
